src/BlockTypes/AbstractInnerBlock.php

```diff
@@ -54,4 +54,13 @@ protected function get_block_type_script( $key = null ) {
 		return parent::get_block_type_script( $key );
 	}
 
+	/**
+	 * Get the frontend style handle for this block type.
+	 *
+	 * @see $this->register_block_type()
+	 * @return string[]|null
+	 */
+	protected function get_block_type_style() {
+		return null;
+	}
```

Comment:
PHPDocs says that the function returns an array of strings or null. Looking at the code, it appears that only null is returned.